kartothek/io_components/metapartition.py

```diff
@@ -340,7 +340,11 @@ def __eq__(self, other):
             return False
 
         for table, meta in self.table_meta.items():
-            if not meta.equals(other.table_meta.get(table, None)):
+            # https://issues.apache.org/jira/browse/ARROW-5873
```

Comment:
Can you add a comment saying this only affects pyarrow versions <= 0.14.0 ?

Reply:
That info is available on the JIRA ticket ;)